document-generator-api/src/test/java/uk/gov/companieshouse/document/generator/api/document/render/RenderDocumentRequestHandlerTest.java

```diff
@@ -36,14 +41,23 @@
 
     private static final String RESOURCE_URI = "/transactions/091174-913515-326060/accounts/xU-6Vebn7F8AgLwa2QHBUL2yRpk=";
 
+    private static final String RESOURCE_ID = "/transactions/091174-913515-326060";
+
     private static final String REQUEST_ID = "requestId";
 
+    private static final int HTTP_CREATED = 201;
```

Comment:
Can we use HttpStatus.SC_CREATED and HttpStatus.SC_INTERNAL_SERVER_ERROR instead of making our own constants.